### ISSUE TITLE
server isn't running when using npx @techdocs/cli serve

### ISSUE THREAD

**Issue by emmaindal** [backstage/techdocs-cli#167]
Trying to preview docs using techdocs cli and seeing an error message.

**Command:**

`npx @techdocs/cli serve`

**Error:**
```
ld: warning: object file (<path>) was built for newer macOS version (11.0) than being linked (10.13)
info: Starting mkdocs server.
info: Waiting for mkdocs server to start...
info: Waiting for mkdocs server to start...
info: Waiting for mkdocs server to start...
info: Waiting for mkdocs server to start...
info: Waiting for mkdocs server to start...
info: Waiting for mkdocs server to start...
info: Waiting for mkdocs server to start...
info: Waiting for mkdocs server to start...
info: Waiting for mkdocs server to start...
info: Waiting for mkdocs server to start...
error: mkdocs server did not start. Exiting. Try re-running command with -v option for more details.
info: Serving docs in Backstage at http://localhost:3000/docs/default/component/local/
Opening browser.
```

The browser window opens and waits, then eventually becomes a blank screen. I'm assuming the server isn't running at all.

**Comment by adrianke77** [comment]
Added. Also it seem the command works on another much smaller set of docs i have but not this larger set

**Comment by emmaindal** [comment]
@adrianke77 thanks a lot, can I ask you for a output on only mkdocs serve as well (as that seems to be the one failing)

`npx @techdocs/cli serve:mkdocs --verbose`

**Comment by adrianke77** [comment]
I tried serve:mkdocs, found that it worked, then tried the original command , then found that it worked too. 

Had to reboot my system this morning before i tried the the two commands. If i intentionally occupy port 8000 the error is different, so it's not that. 

Not sure what else to try..

**Comment by iamEAP** [comment]
This should be resolved with `v0.8.5` of the TechDocs CLI!
